canvas_exercise/shapes_game/index.html

```diff
@@ -1,42 +0,0 @@
-<!DOCTYPE html>
```

Comment:
I'm not sure why this file is getting deleted.  Was there something wrong with the git commit?